tests/unit/drm_plugin_test.sh

```diff
@@ -225,16 +233,99 @@ function test_gui_control_remote()
   compare_command_sequence '' "$LINENO" 'expected_cmd_seq' "$output"
 }
 
+function test_gui_control_after_reboot_local()
+{
+  local gui_on_after_reboot_cmd='systemctl set-default graphical.target'
+  local gui_off_after_reboot_cmd='systemctl set-default multi-user.target'
+  local bind_cmd='for i in /sys/class/vtconsole/*/bind; do printf "%s\n" 1 > $i; done; sleep 0.5'
+  local unbind_cmd='for i in /sys/class/vtconsole/*/bind; do printf "%s\n" 0 > $i; done; sleep 0.5'
+  local output
+
+  # Remote
+  remote_parameters['REMOTE_IP']='127.0.0.1'
+  remote_parameters['REMOTE_PORT']='8888'
+  remote_parameters['REMOTE_USER']='juca'
```

Comment:
This looks like something that we want to put in the `setUp()` function.

Reply:
Unfortunatelly that won't work, as the function `setUp` runs before each test, and there are some tests that need another values for `REMOTE_PORT`. I could probably put the `REMOTE_IP` and `REMOTE_USER` in this function, but if i did that, it would be better that i refactor the whole file.